### PR TITLE
Publish experiment config from taskcluster training task (group_logs)

### LINKED ISSUE #521: Publish experiment config from Taskcluster

### PR DESCRIPTION
**Pull request by vrigal**
Based on #598

Closes #521

### CODE REVIEW

**Comment by eu9ene** [comment]
@vrigal it's this issue again #527. Could you remove this line so that we can test it on CI?

https://github.com/mozilla/firefox-translations-training/blob/c3ad3a4837184a2d82d0e4f0bcf0b7a977ed59c0/taskcluster/translations_taskgraph/parameters.py#L83

File: tracking/translations_parser/cli/taskcluster.py [hunk 1]
```diff
@@ -220,6 +231,29 @@ def boot() -> None:
     )
     parser.run()
 
+    if args.publish_group_logs:
```
Comment (eu9ene) [inline tracking/translations_parser/cli/taskcluster.py:234]: The approach looks good to me, but I think we have this code somewhere in the offline Taskcluster publisher. Should we reuse it?

**Comment by eu9ene** [comment]
Depends on #611 to start CI

**Comment by vrigal** [comment]
Config publication fails because no experiment configuration is found in the CI task group (the script is looking at `extra.action.context.input` in task group). Otherwise it should be fine I think.
> Config could not be read at /tmp/tmpepz6spwv/experiments/ru-en/ci_YhDt4D_wTM2xGIDSLKPS-Q/config.yml: 'NoneType' object is not iterable

**Comment by eu9ene** [comment]
Also, the linter fails, you should do ` task lint-fix`

**Review by eu9ene — APPROVED** [review]

**Comment by eu9ene** [comment]
Ok, I tested it from the Train action and the config is being published from the first training task. @vrigal I also moved the publishing before the training starts so that we can see the config right away.

**Pull request merged**